src/Zicht/Bundle/UrlBundle/Aliasing/Listener.php

```diff
@@ -192,7 +192,11 @@ public function onKernelRequest(Event\GetResponseEvent $event)
                         break;
                     case UrlAlias::MOVE:
                     case UrlAlias::ALIAS:
-                        $event->setResponse(new RedirectResponse($url->getInternalUrl(), $url->getMode()));
+                        $redirectUrl = $url->getInternalUrl();
+                        if ($queryString = $request->getQueryString()) {
+                            $redirectUrl = sprintf('%s?%s', $redirectUrl, $queryString);
+                        }
+                        $event->setResponse(new RedirectResponse($redirectUrl, $url->getMode()));
```

Comment:
eigenlijk ben je hier helemaal klaar met deze functie toch? Waarom stappen we er niet gelijk uit hier? Als er later door iemand onderaan wat word toegevoegd kan dat vervelend zijn. (defensive programming)